src/main/java/kr/pickple/back/game/service/GameService.java

```diff
@@ -71,6 +73,22 @@ public GameResponse findAllGameMembers(final Long gameId, final RegistrationStat
         return GameResponse.of(game, getMemberResponses(game, status));
     }
 
+    public List<GameResponse> findGamesByAddress(final String address, final Pageable pageable) {
+        final MainAddressResponse mainAddressResponse = addressService.findMainAddressByAddressStrings(address);
+
+        final Page<Game> games = gameRepository.findByAddressDepth1AndAddressDepth2(
+                mainAddressResponse.getAddressDepth1(),
+                mainAddressResponse.getAddressDepth2(),
+                pageable
+        );
+
+        return games.stream()
+                .map(game -> GameResponse.of(game, game.getMembersByStatus(CONFIRMED).stream()
+                        .map(MemberResponse::from)
+                        .toList()))
+                .toList();
```

Comment:
`p2`

아래 있는 메서드를 재활용하여 사용할 수 있을 것 같습니다!

```suggestion
        return games.stream()
                .map(game -> GameResponse.of(game, getMemberResponses(game, CONFIRMED)))
                .toList();
```

Reply:
넵 재활용하도록 수정했습니다.
8e28d061dd3229172260f78f33f51127d45598d1